src/task.c

```diff
@@ -1124,9 +1127,19 @@ task_handler (gpointer user_data)
               // If neither started nor finished then fetch the task
               restraint_task_status (task, app_data, "Running", NULL, NULL);
               result = G_SOURCE_REMOVE;
-              g_string_printf(message, "** Fetching task: %s [%s]\n", task->task_id, task->path);
-              app_data->fetch_retries = 0;
-              task->state = TASK_FETCH;
+              if(app_data->state == RECIPE_ABORT){
```

Comment:
I don't think you need these changes.

Reply:
Yes if the whole recipe abort would work than it wouldnt be necesary to have this.